src/spu/interface.h

```diff
@@ -91,8 +91,37 @@ class impl final : public SPUInterface {
     static const size_t NSSIZE = 45;
 
     // spu
+    class main_t_variables {
+      public:
+        int s_1, s_2, fa, ns;
+        uint8_t *start;
+        unsigned int nSample;
+        int ch, predict_nr, shift_factor, flags, d, s;
+        int bIRQReturn = 0;
+        int voldiv = 0;
+        int32_t tmpCapVoice1Index = 0;
+        int32_t tmpCapVoice3Index = 0;
+        PCSX::SPU::SPUCHAN *pChannel;
+        void reset() {
+            s_1 = s_2 = fa = ns = 0;
+            start = nullptr;
+            nSample = ch = predict_nr = shift_factor = flags = d = s = bIRQReturn = voldiv = 0;
+            tmpCapVoice1Index = 0;
+            tmpCapVoice3Index = 0;
+            pChannel = nullptr;
+        }
+    } _vars;
     void MainThread();
     void writeCaptureBufferCD(int numbSamples);
+    void mixAllChannels();
```

Comment:
The naming here is a bit all over the place.

- C++ allows for namespacing, there's no need to add more prefixes.
- Some of those names aren't self explanatory, like SBPos_28.
- The general naming convention of the project is dromedaryCases for method names; leftovers code from WayBefore still needs to be converted / cleaned up.

Reply:
Ok, I will fix it.